### ISSUE TITLE
Keyboard Navigation does not work in contact search results

### ISSUE THREAD

**Issue by forivall** [chrisss404/gnome-shell-ext-SkypeNotification#16]
With other search results, I can navigate between them using the arrow keys, however, this doesn't work with the skype contacts.

I'm on Gnome 3.6. If this works fine in newer versions of gnome-shell, feel free to close this bug report.
